modules/install/pages/admin.adoc

```diff
@@ -1,24 +1,161 @@
-= Creating Administration Accounts
+= Managing JBoss EAP
 
-Brief overview of how to create admin users
+== JBoss EAP 8 Management Options
 
-== Creating Administrator Accounts
+* JBoss EAP 8 has multiple subsystems that you can customize to support Jakarta EE application requirements, such as database connectivity, batch processing, and integration tools.
 
-* Talk about the *add-user* script and other ways to create accounts
-* Distinguish between management users and application users
-* CSV file format
-* Accessing the JBoss EAP web console
-* Accessing the JBoss EAP CLI
-* Do not go too much into details about EAP CLI or web console here. Remember there is a separate chapter for EAP config...
+* The subsystem configuration is maintained in the `standalone.xml` file for the standalone mode, and the `domain.xml` file for a managed domain mode.
```

Comment:
You have not yet introduced what a subsystem is. Replace this with generic sounding "Jboss EAP configuration"

Reply:
Updated